src/app/pages/register/register.component.html

```diff
@@ -51,80 +49,103 @@
     >
       <form
         [formGroup]="bundleForm"
-        class="flex w-full max-w-4xl flex-1 flex-col gap-6"
+        class="flex w-full max-w-6xl flex-1 flex-col gap-6"
       >
-        <h1 class="text-3xl font-semibold text-gray-900">Access bundles</h1>
-        <p class="font-light text-gray-500">
+        <h1 class="text-center text-3xl font-semibold text-gray-900">
+          Access bundles
+        </h1>
+        <p class="text-center font-light text-gray-500">
           Request access to a bundle you need for your research.
         </p>
-
-        @if (
-          bundleForm.get("selectedBundle")?.invalid &&
-          bundleForm.get("selectedBundle")?.touched
-        ) {
-          <div class="text-sm text-red-500">
-            Please select a bundle to continue
-          </div>
-        }
-
-        <!-- Bundle selection cards -->
-        @for (bundle of bundles; track bundle.id) {
-          <div
-            tabindex="0"
-            class="flex w-full rounded-md bg-white shadow-md transition-all duration-200"
-            [ngClass]="{
-              'cursor-pointer hover:-translate-y-0.5 hover:shadow-lg':
-                !bundle.disabled,
-              'cursor-not-allowed opacity-50': bundle.disabled,
-              'ring-2 ring-red-500':
-                bundleForm.get('selectedBundle')?.invalid &&
-                bundleForm.get('selectedBundle')?.touched &&
-                !bundleForm.get('selectedBundle')?.value &&
-                !bundle.disabled,
-              'ring-2 ring-blue-500':
-                bundleForm.get('selectedBundle')?.value === bundle.id,
-            }"
-            (click)="!bundle.disabled && selectBundle(bundle.id)"
-            (keyup.enter)="!bundle.disabled && selectBundle(bundle.id)"
-          >
-            <!-- Bundle content -->
-            <div class="flex w-5/6 items-center justify-start border-r p-12">
-              <div class="w-1/6">
-                @for (logo of bundle.logoUrls; track logo) {
-                  <img
-                    [src]="logo"
-                    [alt]="bundle.name + ' logo'"
-                    class="h-16 w-auto"
-                    [class.-ml-4]="bundle.id === 'fungi'"
-                    [class.h-20]="bundle.id === 'fungi'"
-                  />
-                }
-              </div>
-              <div class="flex w-4/6 flex-col gap-6">
+        <div class="mt-6 flex flex-wrap items-stretch justify-center gap-6">
+          <!-- Bundle selection cards -->
+          @for (bundle of bundles; track bundle.id) {
+            <div
```

Comment:
Can these cards be a little bit wider/have a smaller horizontal margin on smaller screens? They look a bit cramped in the screenshot